.github/workflows/debugger_exploration_testing.yml

```diff
@@ -0,0 +1,362 @@
+name: Debugger Exploration Testing
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+jobs:
+  bottle-testsuite-0_12_19:
+    name: Bottle (master)
+    runs-on: ubuntu-latest
+    env:
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: bottle
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: bottlepy/bottle
+          ref: master
+          path: bottle
+      - name: Install Dependencies
+        run: sudo apt-get install -y libev-dev
+      - name: Test dependencies (Server back-ends and template engines)
+      # Taken from install script inside of .github/workflows of test suite (https://github.com/bottlepy/bottle/blob/master/.github/workflows/run_tests.yml)
```

Comment:
nit: probably worth getting a perma (commit-hash) link here

Reply:
This source was adapted from a verbatim copy of `test_frameworks.yml`. Perhaps we can address these issues in a clean-up PR? Or perhaps consolidate the tests in a single run by enabling exploration tests in `test_frameworks.yml`?

Reply:
> Or perhaps consolidate the tests in a single run by enabling exploration tests in test_frameworks.yml?

Let's explore this option briefly, I'm ok to not do this and merge this as-is if it is too much overhead to merge the suites